packages/cashier/src/pages/payment-agent/payment-agent-signup/cancel-wizard-dialog.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { Dialog } from '@deriv/components';
+import { localize } from '@deriv/translations';
+
+type TCancelWizardDialogProps = {
+    onConfirm: () => void;
+    onCancel: () => void;
+    is_visible: boolean;
+};
+
+const CancelWizardDialog = ({ onConfirm, onCancel, is_visible }: TCancelWizardDialogProps) => {
+    return (
+        <Dialog
+            title={localize('Are you sure you want to cancel?')}
+            confirm_button_text={localize('Yes, cancel')}
+            cancel_button_text={localize('No, take me back')}
+            onConfirm={() => onConfirm()}
```

Comment:
```suggestion
            onConfirm={onConfirm}
```

Reply:
@hamid-deriv I think both `onConfirm` and `onCancel` can be handled inside this component and there is no need to get them as a prop 🤔

Reply:
@farzin-fs `visible` state will be defined in the wizard component that this component will be used there so that the wizard component will handle the state on close. Also, there are actions in the wizard that should be triggered in the wizard, so I think that it would be easier and more readable to handle the confirm action in the wizard, too.